README.md

```diff
@@ -1,18 +1,180 @@
-# Workleap.Eventgrid.Emulator
+# Workleap Azure Event Grid Emulator
 
+This Docker image provides a local emulator for **Azure Event Grid**, which can be used for testing and development purposes.
+
+## Prerequisites
+
+Before you can use the EventGridEmulator Docker image, you need to have Docker installed on your machine. You can download Docker from the official website: https://www.docker.com/get-started
 
 ## Getting started
 
-|Package| Download Link                                                                                        | Description                                                                |
-|----|------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------|
-|    |  |   |
+To use the EventGridEmulator Docker image, you can follow these steps:
 
+1. Create a development certificate ([for more info](https://github.com/FiloSottile/mkcert)):
 
-## Building, releasing and versioning
+    ```powershell
+    .\devtools\Install-Certificate.ps1
+    ```
 
-The project can be built by running `Build.ps1`.
+    You should see something like:
+    ```
+    Installing localhost certificate...
+    The local CA is already installed in the system trust store! 👍
+
+
+    Created a new certificate valid for the following names 📜
+    - "localhost"
+    - "127.0.0.1"
+    - "::1"
+    - "host.docker.internal"
+
+    The certificate is at "C:\Users\<user>\.eventgridemulator\localhost.crt" and the key at "C:\Users\<user>\.eventgridemulator\localhost.key" ✅
+
+    It will expire on <dd mmmm yy> 📅
+
+    Certificate installed successfully.
+    ```
+
+2. Option a) Using docker-compose file (**recommended**):
+   
+   Create a file named `docker-compose.yaml`.
+
+    ```yaml
+    version: '3.4'
+
+    services:
+        eventgridemulator:
+            image: eventgridemulator
+            environment:
+                - ASPNETCORE_ENVIRONMENT=Development
+                - ASPNETCORE_URLS=https://+:443;http://+:8080
+                - KESTREL__CERTIFICATES__DEFAULT__PATH=/etc/ssl/certs/localhost.crt
+                - KESTREL__CERTIFICATES__DEFAULT__KEYPATH=/etc/ssl/certs/localhost.key
+            ports:
+                - "6500:443"
+            extra_hosts:
+                - "host.docker.internal:host-gateway"
+            volumes:
+                - "~/.eventgridemulator:/etc/ssl/certs"
+    ``` 
+    Run docker compose:
+    ```
+    docker compose up
+    ```
+
+    Option b) Using Docker Cli (*not recommended*):
```

Comment:
If it's not recommended then I would suggest to just remove it and mention only option A